code/script.js

```diff
@@ -253,38 +257,53 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+
+  board.style.display = "flex"
+  winOrLose.style.display = "none"
   // What else should happen when we start the game?
+  setSecret()
+  generateBoard()
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-
+  const value = questions.value
+  
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
+      value: value,
       category: category,
     }
   } else if (category === 'eye color') {
-    // Set this up your self
-  } else if (category === 'accessories') {
     currentQuestion = {
-      //attribute: ,
-      // 👆 this is the property of the booleans such as smoke, glasses and hat. add the value from the input here
-      value: true, // we're asking if this person wears a hat for exaple, so always true in the question.
+      attribute: 'eyeColor',
+      value: value,
       category: category,
     }
+  } else if (category === 'accessories') {
+    currentQuestion = {
+      attribute: value,
+      value: true,
+      category: category,
+    } 
+    
   } else if (category === 'other') {
-    // Set this up your self (should be same structure as above)
-  }
+    currentQuestion = {
+      attribute: 'smoker',
```

Comment:
Since `smoker` is the only other attribute this is completely fine, but for the scalability, you could use the value here as well (if you, later on, decide to add more attributes to the people):
`attribute: value,`